tests/tracer/test_propagation.py

```diff
@@ -368,6 +373,328 @@ def test_get_wsgi_header(tracer):
     assert get_wsgi_header("x-datadog-trace-id") == "HTTP_X_DATADOG_TRACE_ID"
 
 
+# for testing with other propagation styles
+TRACECONTEXT_HEADERS_VALID_BASIC = {
+    _HTTP_HEADER_TRACEPARENT: "00-4bf92f3577b34da6a3ce929d0e0e4736-00f067aa0ba902b7-01",
+    _HTTP_HEADER_TRACESTATE: "dd=s:2;o:rum",
+}
+
+TRACECONTEXT_HEADERS_VALID = {
+    _HTTP_HEADER_TRACEPARENT: "00-4bf92f3577b34da6a3ce929d0e0e4736-00f067aa0ba902b7-01",
+    _HTTP_HEADER_TRACESTATE: "dd=s:2;o:rum;t.dm:-4;t.usr.id:baz64,congo=t61rcWkgMzE",
+}
+
+
+@pytest.mark.parametrize(
+    "name,sampling_priority_tp,sampling_priority_ts,expected_sampling_priority",
```

Comment:
We can use the `id` kwarg to `pytest.mark.parameterize` instead of `name`, however I think we can drop it entirely as I don't think there is an easy human readable gloss for each parameter tuple.

https://docs.pytest.org/en/7.1.x/example/parametrize.html#different-options-for-test-ids